lib/getMapfile.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+const path = require('path')
+
+module.exports = function getMapfile(p) {
+  const ext = path.extname(p)
+  const base = path.basename(p)
+  return p.replace(base, base.replace(ext, `${ext}.map`))
```

Comment:
Would this not be effectively the same? Or am I missing something.

```suggestion
  return p + '.map'
```